ndbench-core/src/main/java/com/netflix/ndbench/core/config/IConfiguration.java

```diff
@@ -89,7 +96,25 @@ default int getNumReaders() {
     @DefaultValue("100")
     int getReadRateLimit();
 
+
     @DefaultValue("100")
+    @PropertyName(name=WRITE_RATE_LIMIT)
     int getWriteRateLimit();
 
+
+    @DefaultValue("false")
+    boolean isAutoTuneEnabled();
+
+
+    @DefaultValue("60")
```

Comment:
regarding java.util.*  -- i ran optimize imports.. i should have done that just on files i changed.  the result you have is what IDEA  does instead of importing the following individually:

import java.util.HashMap;
import java.util.Map;
import java.util.Random;
i